Stellar-ledger-entries.x

```diff
@@ -499,6 +500,17 @@ struct ContractDataEntry {
     SCVal val;
 };
 
+struct ContractCodeEntry {
+    Hash hash;
+    opaque code<SCVAL_LIMIT>;
+    union switch (int v)
+    {
+    case 0:
+        void;
+    }
+    ext;
+};
```

Comment:
We should keep extension points at the front of types, and to use the predefined type for them. The reason for putting it at the front is so that we defer the decision on whether the type should be a chainable extension style, or a full body union style type. Here's an example of where we do that:
https://github.com/stellar/stellar-xdr-next/blob/48d5e17ae63bba0aa9725cd9d18d7438f44c07b1/Stellar-ledger.x#L394-L398

Iirc this was something @stanford-scs encouraged us to start doing in a recent protocol release.